modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DefaultDLViewFileVersionDisplayContext.java

```diff
@@ -200,7 +205,10 @@ public List<DDMStructure> getDDMStructures() throws PortalException {
 			DLFileVersion dlFileVersion =
 				(DLFileVersion)_fileVersion.getModel();
 
-			_ddmStructures = dlFileVersion.getDDMStructures();
+			_ddmStructures = ListUtil.filter(
+				dlFileVersion.getDDMStructures(),
+				ddmStructure ->
+					ddmStructure.getType() != DDMStructureConstants.TYPE_AUTO);
```

Comment:
When you create a Document type, the type of the ddmStructure is updated to `TYPE_AUTO` , so we are hiding the metadata on the info_panel not only on the google/videos but all

https://github.com/liferay-lima/liferay-portal/blob/47319863a397b61f650554f512657e0f7f6bc34f/modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/service/DataEngineDLFileEntryTypeLocalServiceWrapper.java#L101